plugins/ata/fu-ata-device.c

```diff
@@ -108,7 +109,7 @@ fu_ata_device_get_string (const guint16 *buf, guint start, guint end)
 
 	/* remove whitespace before returning */
 	if (str->len > 0) {
-		g_strchomp (str->str);
+		g_strstrip (str->str);
```

Comment:
Leading spaces too?

Reply:
Yep, without it I was seeing GUID `STORAGE-DELL-  013856` built

Reply:
Fair enough.